### DISCUSSION TITLE
Clone function malfunction

### DISCUSSION THREAD

**Discussion by maksbrzezinski** [OrcaSlicer/OrcaSlicer#5921] [Dev]
Hi everyone, 

I have the newest 2.1.1 V of OrcaSlicer and I have tried the Clone function today to find out that it doesn't work. Anything over "1" is ignored and only one copy of a file is being created.

**Comment by maksbrzezinski** [comment]
In fact, after some time it started copying as expected. Seems to be a temporary bad mood.

**Comment by lukicsl** [comment]
It clones, though just in place, you have to distribute after cloning:
![image](https://github.com/SoftFever/OrcaSlicer/assets/23736716/674f956d-9ec7-4fbf-877f-eb5716bc5301)

**Reply by foreachthing** [reply]
This "issue" is a bit nicer solved in PrusaSlicer, I think. It moves the clones in XY while placing. This makes it more obvious that you have to arrange everything again.

**Reply by maksbrzezinski** [reply]
I do realize it copies files in the same place when working correctly. I've been using Orca for about 6 months now. The issue I faced was literally not copying at all after typing anything more than 1 in the copy pop-up.
